app/src/features/mocks/screens/mocksList/components/MocksList/components/MocksTable/hooks/useMocksTableColumns.tsx

```diff
@@ -38,47 +58,81 @@ export const useMocksTableColumns = ({
       ),
       dataIndex: "name",
       ellipsis: true,
-      width: 320,
-      render: (_: any, record: RQMockMetadataSchema) => {
-        return (
-          <div
-            className="mock-details-container"
-            onClick={(e) => {
-              e.preventDefault();
-              e.stopPropagation();
-              handleNameClick(record.id, record.isOldMock);
-            }}
-          >
-            <div className="mock-name-container">
-              <span
-                className="mock-type"
-                style={{
-                  color:
-                    record?.type === MockType.API
-                      ? REQUEST_METHOD_COLORS[record.method]
-                      : fileTypeColorMap[record.fileType],
+      width: 300,
+      render: (_: any, record: RQMockSchema) => {
+        const isCollection = isRecordMockCollection(record);
+
+        return isCollection ? (
+          <div className="mock-collection-details-container">
+            <span className="collection-icon">
+              <MdOutlineFolder />
+            </span>
+
+            <Typography.Text ellipsis={true} className="mock-collection-name">
+              {record.name}
+            </Typography.Text>
+
+            <Typography.Text ellipsis={true} className="mock-collection-description">
```

Comment:
I think Collection description should be in info icon as per design . Please check this with designs